mlflow/pytorch/_lightning_autolog.py

```diff
@@ -38,10 +61,12 @@
 _logger = logging.getLogger(__name__)
 
 _pl_version = Version(pl.__version__)
-if _pl_version < Version("1.5.0"):
+if _pl_version < Version("1.5.0") and PYTORCH_LIGHTNING_LEGACY_NAMESPACE is True:
     from pytorch_lightning.core.memory import ModelSummary
-else:
+elif PYTORCH_LIGHTNING_LEGACY_NAMESPACE is True:
     from pytorch_lightning.utilities.model_summary import ModelSummary
+else:
+    from lightning.pytorch.utilities.model_summary import ModelSummary
```

Comment:
Does lightning < 1.5.0 also imports from this namespace?
It would be clearer if we use `if PYTORCH_LIGHTNING_LEGACY_NAMESPACE, else` as the outer check and check version inside.

Reply:
Does this comment apply `Does lightning < 1.5.0 also imports from this namespace?`?